core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -33,6 +33,7 @@
 #include <sm/optee_smc.h>
 #include <sm/teesmc_opteed_macros.h>
 #include <sm/teesmc_opteed.h>
+#include <kernel/spinlock.h>
```

Comment:
Not needed, definitely. Not sure how I could get this wrong but anyway, I'll remove the line.